contracts/registries/IPAssetRegistry.sol

```diff
@@ -87,19 +87,29 @@ contract IPAssetRegistry is
         address tokenContract,
         uint256 tokenId
     ) external whenNotPaused returns (address id) {
-        id = _register({ chainid: chainid, tokenContract: tokenContract, tokenId: tokenId });
+        id = _register({
+            chainid: chainid,
+            tokenContract: tokenContract,
+            tokenId: tokenId,
+            registerFeePayer: msg.sender
```

Comment:
Similar to the previous question, should we add an extra `registerFeePayer` parameter to the `register` function instead of relying on `msg.sender`?